### PR TITLE
Add missing return $this php docs

### PR DESCRIPTION
**Pull request by blankse**

### CODE REVIEW

File: models/DataObject/ClassDefinition/Data/Localizedfields.php [hunk 1]
```diff
@@ -883,16 +895,6 @@ public function getMaxTabs(): int
         return $this->maxTabs;
     }
 
-    public function setLabelWidth(int $labelWidth): void
```
Comment (fashxp) [inline models/DataObject/ClassDefinition/Data/Localizedfields.php:886]: why is that removed here? 
at least it doesn't match with PRs description...

File: models/DataObject/ClassDefinition/Helper/VarExport.php [hunk 2]
```diff
@@ -47,9 +47,6 @@ public static function __set_state(array $data): static
         return $obj;
     }
 
-    /**
```
Comment (fashxp) [inline models/DataObject/ClassDefinition/Helper/VarExport.php:50]: why is `$this` removed here? also in other places?

**Review by fashxp — CHANGES_REQUESTED** [review]

File: models/DataObject/ClassDefinition/Helper/VarExport.php (hunk 2 above)
Comment (blankse) [reply models/DataObject/ClassDefinition/Helper/VarExport.php:50]: Because the method is also defined in the parent class. The php docs are inherit.

File: models/DataObject/ClassDefinition/Data/Localizedfields.php (hunk 1 above)
Comment (blankse) [reply models/DataObject/ClassDefinition/Data/Localizedfields.php:886]: Because the method is also defined in the LabelTrait trait.

**Review by fashxp — APPROVED** [review]
